### PR TITLE
feat: added DownloadDir :sparkles:

### PR DESCRIPTION
**Pull request by Onibenjo**

### CODE REVIEW

File: android/src/main/java/com/alpha0010/FileAccessModule.kt [hunk 1]
```diff
@@ -25,7 +26,8 @@ class FileAccessModule(reactContext: ReactApplicationContext) : ReactContextBase
       "CacheDir" to reactApplicationContext.cacheDir.absolutePath,
       "DatabaseDir" to reactApplicationContext.getDatabasePath("FileAccessProbe").parent,
       "DocumentDir" to reactApplicationContext.filesDir.absolutePath,
-      "MainBundleDir" to reactApplicationContext.applicationInfo.dataDir
+      "MainBundleDir" to reactApplicationContext.applicationInfo.dataDir,
+      "DownloadDir", to Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS).getAbsolutePath()
```
Comment (alpha0010) [inline android/src/main/java/com/alpha0010/FileAccessModule.kt:30]: Documentation says [`getExternalStoragePublicDirectory()`](https://developer.android.com/reference/android/os/Environment#getExternalStoragePublicDirectory(java.lang.String)) is deprecated, and recommends [`getExternalFilesDir()`](https://developer.android.com/reference/android/content/Context#getExternalFilesDir(java.lang.String)).  The latter however, restricts file visibility to only the current app.  Does that still meet your needs for access to the downloads directory?

**Comment by alpha0010** [comment]
Does `cpExternal()` from master solve this for you?  (`npm install alpha0010/react-native-file-access` should pull the latest unpublished code.)

**Comment by alpha0010** [comment]
Thanks for you contribution.  Implemented feature differently in v0.6.0.

**Comment by Onibenjo** [comment]
I'll try the master branch, thanks

**Pull request closed without merging**